packages/components/inputs/select-utils/src/create-select-styles.ts

```diff
@@ -285,45 +315,50 @@ const menuPortalStyles = (props: TProps) => (base: TBase) => ({
   zIndex: props.menuPortalZIndex,
 });
 
-const multiValueStyles = () => (base: TBase) => {
-  return {
-    ...base,
-    height: designTokens.sizeHeightTag,
-    backgroundColor: designTokens.backgroundColorForTag,
-    padding: '0',
+const multiValueStyles =
+  (props: TProps & { isNewTheme: boolean }) => (base: TBase) => {
+    return {
+      ...base,
+      height: designTokens.sizeHeightTag,
+      backgroundColor: designTokens.backgroundColorForTag,
+      padding: '0',
+      border: props.isNewTheme && `1px solid ${designTokens.colorNeutral85}`,
+    };
   };
-};
 
-const multiValueLabelStyles = (props: TProps) => (base: TBase) => {
-  return {
-    ...base,
-    fontSize: designTokens.fontSizeSmall,
-    color: (() => {
-      if (props.isDisabled) return designTokens.fontColorForInputWhenDisabled;
-      if (props.isReadOnly) return designTokens.fontColorForInputWhenReadonly;
-      return base.color;
-    })(),
-    padding: `${designTokens.spacing10} ${designTokens.spacing20}`,
-    borderRadius: `${designTokens.borderRadiusForTag} 0 0 ${designTokens.borderRadiusForTag}`,
-    border: `1px ${designTokens.borderColorForTag} solid`,
-    borderWidth: '1px 0 1px 1px',
+const multiValueLabelStyles =
+  (props: TProps & { isNewTheme: boolean }) => (base: TBase) => {
+    return {
+      ...base,
+      fontSize: designTokens.fontSizeSmall,
+      color: (() => {
+        if (props.isDisabled) return designTokens.fontColorForInputWhenDisabled;
+        if (props.isReadOnly) return designTokens.fontColorForInputWhenReadonly;
+        return base.color;
+      })(),
+      padding: `${designTokens.spacing10} ${designTokens.spacing20}`,
+      borderRadius: `${designTokens.borderRadiusForTag} 0 0 ${designTokens.borderRadiusForTag}`,
+      border: props.isNewTheme
+        ? 'none'
+        : `1px ${designTokens.borderColorForTag} solid`,
+      borderWidth: '1px 0 1px 1px',
 
-    '&:last-child': {
-      borderRadius: designTokens.borderRadiusForTag,
-      borderWidth: '1px',
-    },
+      '&:last-child': {
+        borderRadius: designTokens.borderRadiusForTag,
+        borderWidth: '1px',
+      },
+    };
   };
-};
 
 const multiValueRemoveStyles =
-  (props: TProps) => (base: TBase, state: TState) => {
+  (props: TProps & { isNewTheme: boolean }) => (base: TBase, state: TState) => {
     return {
       ...base,
       borderColor: designTokens.borderColorForTag,
       padding: `0 ${designTokens.spacing10}`,
       borderRadius: `0 ${designTokens.borderRadiusForTag} ${designTokens.borderRadiusForTag} 0`,
-      borderStyle: 'solid',
-      borderWidth: '1px',
+      borderStyle: !props.isNewTheme && 'solid',
+      borderWidth: !props.isNewTheme && designTokens.borderWidth1,
```

Comment:
We need alternative values for the use case when `props.isNewTheme` is _false_.

```suggestion
      borderStyle: !props.isNewTheme ? 'solid' : 'none',
      borderWidth: !props.isNewTheme ? designTokens.borderWidth1 : '0',
```

Reply:
Thank you.
All updated here: https://github.com/commercetools/ui-kit/pull/2373/commits/9123d3cfb0ba7a0b5371e8cd341a0e679b265e4d